### DISCUSSION TITLE
It's been 3 days since approved but didn't got the benefit till now

### DISCUSSION THREAD

**Discussion by manishgupta2026** [community/community#128629] [GitHub Education]
### Hi everyone,

Hey team , 
It's been **3** days since I got **approved for GitHub students pack** but **offers are not available till now**_.
Please look into the matter and solve it please.
It would be very helpful for me.
Thanks

**Comment by davevad93** [comment]
Hi @manishgupta2026 ,

What's the state of your application? The benefits are fully available when it says "`Congrats! Your request for @manishgupta2026 was approved and your benefits are now available`" if it states "`Your benefits will be available soon`" it means you still have to wait a few days.

**Reply by manishgupta2026** [reply]
Yep it's says " your benefits will be available soon " but it's been 3 days with same message

**Reply by davevad93** [reply]
Hey @manishgupta2026 don't worry, the waiting time may vary depending on different factors so be patient, I think that in 1 week at most you will have your benefits. Take a look at this discussion #128457 , this user had to wait almost 10 days.

**Reply by manishgupta2026** [reply]
Hopefully 🙂

**Comment by ishubtripathi** [comment]
hii @manishgupta2026 
my the discussion #55559 will help you to get your answer.
read by following link - https://github.com/orgs/community/discussions/55559
